src/convenience/keyboard.ts

```diff
@@ -112,6 +133,62 @@ export class Keyboard {
     webApp(text: string, url: string) {
         return this.add({ text, web_app: { url } });
     }
+    /**
+     * Make the current keyboard selective.
+     * See https://grammy.dev/plugins/keyboard.html#selectively-send-custom-keyboards
+     * for more details.
+     *
+     * Keyboards are non-selective by default,
+     * use this function to enable it(without any param or pass a `true`).
```

Comment:
```suggestion
     * use this function to enable it (without any parameters or pass a `true`).
```

Reply:
Wouldn't it be _pass `true`_ instead of _pass a `true`_?

Reply:
Correct. Didn't spot that.

Reply:
It's done.